src/network/networkctl.c

```diff
@@ -3190,6 +3201,8 @@ static int run(int argc, char* argv[]) {
 
         sigbus_install();
 
+        arg_ask_password = isatty(STDIN_FILENO);
```

Comment:
Hmm? Is this needed? We don't do this in other tools?

Reply:
Yeah, I am lazy about adding --ask-password=BOOL option to networkctl and resolvectl.
Do you think we should add the option to them? If so, I will add it.

Reply:
It's probably OK, since this hasn't been requested in any RFE and is relatively easy to add. But maybe explain the situation in comments?

Reply:
I changed my mind, and added --no-ask-password like other tools.